framework/Optimizers/Optimizer.py

```diff
@@ -580,13 +585,15 @@ def handleFailedRuns(self,failedRuns):
     if len(failedRuns)>0:
       self.raiseAWarning('There were %i failed runs!  Run with verbosity = debug for more details.' %(len(failedRuns)))
       for run in failedRuns:
-        metadata = run.getMetadata()
-        self.raiseADebug('  Run number %s FAILED:' %run.identifier,run.command)
+        ## FIXME: run.command no longer exists, so I am only outputting the
+        ## run's identifier, also taking out the metadata info for now.
+        self.raiseADebug('  Run number %s FAILED:' %run.identifier)
         self.raiseADebug('      return code :',run.getReturnCode())
-        if metadata is not None:
-          self.raiseADebug('      sampled vars:')
-          for v,k in metadata['SampledVars'].items():
-            self.raiseADebug('         ',v,':',k)
+        # metadata = run.getMetadata()
+        # if metadata is not None:
+        #   self.raiseADebug('      sampled vars:')
+        #   for v,k in metadata['SampledVars'].items():
+        #     self.raiseADebug('         ',v,':',k)
```

Comment:
This could probably be left in the code on second thought. Given other development efforts, it is probably best for me not to unnecessarily touch parts of the code like this. I will unroll this change when I receive comments from everyone else as long as it does not break anything (it should not).